### ISSUE TITLE
Man pages for "wm?" contain "@WML_VERSION@" instead of the version number

### ISSUE THREAD

**Issue by xtaran** [thewml/website-meta-language#22]
Currently at least the man pages for wml, wmk, wmd and wmb contain `@WML_VERSION@` instead of the WML version number. `man ./wml_frontend/wml.1` (after `cmake . && make` in `src/`) yields:

```
WML(1)                                 EN Tools                                 WML(1)

NAME
       WML - Website META Language

VERSION
       @WML_VERSION@

SYNOPSIS
[…]
```

I'm not yet sure what's the reason for this, but I suspect that the POD in `wm?.src`is taken before any replacement took part.

Seems to be buried somewhere down in the CMake includes.

**Comment by shlomif** [comment]
Thanks for the report!

**Comment by shlomif** [comment]
Should be fixed in 2f2a8aea6d9a0fc2232a0d7bae8f537662cbe7a9 - please test.

**Comment by xtaran** [comment]
Looks fine, thanks!
